CONTRIBUTING.rst

```diff
@@ -48,6 +48,10 @@ Then you can run the testsuite with::
 
     py.test
 
+- Note: Werkzeug must also be installed for all tests to pass::
```

Comment:
Ah, good point... all good, that gives me a chance to fix my author/committer email mismatch anyways.